atlashdf/osm_resolvegeom.cpp

```diff
@@ -199,8 +198,108 @@ void _resolve_osm_linestrings(std::string inputfile) {
   }
 }
 
+void _resolve_osm_relations(std::string inputfile) {
+  std::cout << "Resolving relations..." << std::endl;
+  H5Easy::File file(inputfile, H5Easy::File::ReadWrite);
+
+  // load linestring indices
+  auto ways = H5Easy::load<std::vector<uint64_t>>(file, "/osm/ways");
+  std::cout << "Way count: " << ways.size() << std::endl;
+
+  std::map<uint64_t, uint64_t> osm2row;
+  for (size_t i = 0; i < ways.size(); i++) osm2row[ways[i]] = i;
+  ways.clear();
+
+  // create datasets
+  if (!file.getGroup("osm").exist("parts")) {
+    HighFive::DataSpace ds =
+        HighFive::DataSpace({0, 2}, {HighFive::DataSpace::UNLIMITED, 2});
+    HighFive::DataSetCreateProps props;
+    props.add(HighFive::Chunking(std::vector<hsize_t>{1024 * 1024, 2}));
+    props.add(HighFive::Deflate(9));
+
+    file.getGroup("osm").createDataSet("parts", ds,
+                                       HighFive::AtomicType<double>(), props);
+    file.getGroup("osm").createDataSet("parts_idx", ds,
+                                       HighFive::AtomicType<uint32_t>(), props);
+  }
+
+  // open created datasets
```

Comment:
Are these parts tables efficient and needed? I can't remember them in the specifications? Let us talk in a review meetig